cmd/server-mux.go

```diff
@@ -244,6 +244,17 @@ func initListeners(serverAddr string, tls *tls.Config) ([]*ListenerMux, error) {
 	return listeners, nil
 }
 
+// Is network connection really closed?.
+func isCloseConnectionErr(err error) bool {
+	if nerr, ok := err.(*net.OpError); ok {
+		nmsgMatch := strings.EqualFold(nerr.Err.Error(), "use of closed network connection")
```

Comment:
instead of doing this - i.e trying to match the errors' String(), better would be to have a struct field in `ServerMux.graceFullyShutdown` which can be set when the server is shutdown so that it can be checked here if error needs to be printed. String matching error's String() does not look clean unless there is no alternative.